site/pages/create-disruption.tsx

```diff
@@ -1,5 +1,211 @@
+import DateSelector from "components/DateSelector";
+import TimeSelector from "components/TimeSelector";
+import { BaseLayout } from "components/layout/Layout";
+
+const title = "Create Disruptions";
+const description = "Create Disruptions page for the Create Transport Disruptions Service";
+
+enum Reason {
```

Comment:
if these are all mapping to the exact same value as a string, would it make sense for this to just be a string array? And then lower down, instead of using Object.keys , we can just map over the array? Im not sure what value we get from using an enum here.

Reply:
Although in saying that, maybe the benefit is because we shouldnt be using camelCase values as what the user selects - should it be `roadWorks = "Road works",`

Reply:
I'd maybe approach it slightly differently and use the types defined in the shared-ts folder here, something like:
```
const reasons: {
    value:
        | siriTypes.MiscellaneousReason
        | siriTypes.PersonnelReason
        | siriTypes.EnvironmentReason
        | siriTypes.EquipmentReason;
    display: string;
}[] = [
    {
        value: siriTypes.MiscellaneousReason.accident,
        display: "Accident",
    },
    {
        value: siriTypes.PersonnelReason.staffAbsence,
        display: "Staff absence",
    },
];
```

Reply:
Updated the key value pairs and code to access key and value. The enum was used because the XML element for Reason is also an enum and we could reuse the shared type when we have all the values

Reply:
I think an array makes more sense here, the enums in the shared types won't be changing so we can still make use of those now to give us some type safety for the array, when Steve gives us the actual values we can just update the list